Source/WebCore/animation/ViewTimeline.cpp

```diff
@@ -74,13 +78,56 @@ Ref<ViewTimeline> ViewTimeline::createFromCSSValue(Style::BuilderState& builderS
     return adoptRef(*new ViewTimeline(nullAtom(), axis, { startInset, endInset }));
 }
 
-// FIXME: compute offset values from options.
+static std::optional<Length> lengthForInset(std::variant<RefPtr<CSSNumericValue>, RefPtr<CSSKeywordValue>> inset)
+{
+    // TODO: Figure out how to convert CSSNumericValue to Length
```

Comment:
Isn't that what `lengthForInsetCSSValue()` is for?

Reply:
No that one does CSSPrimitiveValue -> Length. For this I need to understand how to go from CSSNumericValue -> Length (and how to trigger this codepath via javascript).